draft-ietf-tls-tls13.md

```diff
@@ -2221,6 +2226,25 @@ Note: In versions of TLS prior to TLS 1.3, this extension was named
 "elliptic_curves" and only contained elliptic curve groups. See
 {{RFC4492}} and {{I-D.ietf-tls-negotiated-ff-dhe}}.
 
+All clients MUST send a valid "supported_groups" extension containing
+at least one group for each key exchange algorithm (currently
+DHE and ECDHE) for which it offers a cipher suite.
```

Comment:
Well, I sort of consider however the PSK got shared out-of-band as the key exchange. PSK suites just use it.

Just adding "ephemeral" would make it stand completely apart rather easily, I guess.